controllers/endpoints-controllers.js

```diff
@@ -0,0 +1,6 @@
+const { fetchEndpoints } = require('../models/endpoints-models');
+
+exports.getEndpoints = (req, res, next) => {
+  const endpoints = fetchEndpoints();
+  res.status(200).send({ endpoints });
+};
```

Comment:
You don't need a model for this one as we are not querying the database. You can require the json into the controller and send it straight back.